PWG/EMCAL/EMCALtasks/AliEmcalCorrectionComponent.h

```diff
@@ -136,12 +138,14 @@ class AliEmcalCorrectionComponent : public TNamed {
   TString                fBasePath;                       ///< Base folder path to get root files
   TString                fCustomBadChannelFilePath;       ///< Custom path to bad channel map OADB file
 
+  Bool_t 		 fDoMergedBCs;			  ///< flag to use one histo for all BCs
```

Comment:
This is only used in the time calibration, so it should be defined in the time calibration component